packages/neon-nep5/src/plugin.ts

```diff
@@ -1,4 +1,4 @@
 import * as abi from "./abi";
-import { getToken, getTokenBalance } from "./main";
+import { getToken, getTokenBalance, getTokens } from "./main";
 
-export { abi, getToken, getTokenBalance };
+export { abi, getToken, getTokens, getTokenBalance };

```

Comment:
I apologise for my error but since you are touching this file, could you also include the export for `getTokenBalances`? I did not realise that was not exported.

Reply:
Done!